packages/viz/src/lib/style/helpers/utils.ts

```diff
@@ -9,23 +9,38 @@ export function getUpdateTriggers(accessorFunction: Record<string, unknown>) {
   };
 }
 
+// export function getColors(
+//   colorProperty: string | string[],
+//   colorLength: number
+// ) {
+//   if (typeof colorProperty === 'string') {
+//     const { colors: rgbaColors, othersColor } = getColorPalette(
+//       colorProperty.toUpperCase(),
+//       colorLength
+//     );
+
+//     return {
+//       rgbaColors: rgbaColors.map(hexToRgb),
+//       othersColor: othersColor ? hexToRgb(othersColor) : undefined
+//     };
+//   }
+
+//   return { rgbaColors: (colorProperty as string[]).map(hexToRgb) };
+// }
+
 export function getColors(
   colorProperty: string | string[],
   colorLength: number
 ) {
   if (typeof colorProperty === 'string') {
-    const { colors: rgbaColors, othersColor } = getColorPalette(
+    const { colors } = getColorPalette(
       colorProperty.toUpperCase(),
       colorLength
```

Comment:
We should probably manage here any possible exception, trying to get more colors than existing ones in the palette